packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx

```diff
@@ -8,10 +8,8 @@ import {
 import { getColorFromString } from '../../utilities/color/getColorFromString';
 import { GridCell } from '../../utilities/grid/GridCell';
 import { IGridCellProps } from '../../utilities/grid/GridCell.types';
-import { IButtonClassNames } from '../Button/BaseButton.classNames';
-import { getStyles as getActionButtonStyles } from '../Button/ActionButton/ActionButton.styles';
-import { mergeStyleSets, ITheme } from '../../Styling';
 import { classNamesFunction } from '../../Utilities';
+import { getColorPickerGridCellButtonClassNames as getCustomClassNamesForColorPickerButton } from './ColorPickerGridCell.classNames';
```

Comment:
I think it would be better to just leave this function in the current file (but as a standalone function instead of a private member) rather than making a new `.classNames.ts`--the `.classNames.ts` files seem to be used for a slightly different purpose and only by components which use an old styling approach we're moving away from.

Reply:
Sure....